includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
And I was also thinking about resolving [this TODO](https://github.com/Wikia/app/blob/dev/extensions/wikia/GlobalFooter/scripts/GlobalFooter.js#L43).